.github/workflows/Bootstrap.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           version: 1
       - run: |
-          julia  -e 'using Pkg; Pkg.add("Latexify"); Pkg.develop(path="."); include("src/create_README.jl")'
+          julia  -e 'using Pkg; Pkg.add("Latexify"); Pkg.develop(path="."); include("📜/👢🎀.jl")'
```

Comment:
Looks like this file is missing.

Reply:
It is present in the :hand: branch, will merge.